src/app/pages/home/prysmatic-home.page.component.html

```diff
@@ -0,0 +1,371 @@
+                </table>
+            </div>
+            <div class="column">
+                <div class="w-full">
+                    <figure class="image is-square">
+                        <img src="./assets/product/prysm.gif" class="rounded shadow-lg" alt="product prysm">
+                    </figure>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section bg-grey-light ">
+    <div class="container bg-grey-light ">
+        <h1 class="title">Sponsors and Grants</h1>
+        <h2 class="subtitle">
+            Thank you to all who have contributed to Prysmatic Labs; we are eternally grateful.
+        </h2>
+
+        <div class="columns m-auto is-multiline is-mobile">
+            <div class="column text-center is-half-mobile ">
+                <a href="https://www.aragon.one" alt="argon-one">
+                    <img src="./assets/sponsors/aragonnest.svg" alt="aragonnest" width="100px" />
+                </a>
+            </div>
+            <div class="column text-center is-half-mobile">
+                <a href="https://district0x.io" alt="district0x">
+                    <img src="./assets/sponsors/district0x.svg" alt="district0x" width="100px" />
+                </a>
+            </div>
+            <div class="column text-center">
+                <a href="https://ecf.network" alt="ECF">
+                    <img src="./assets/sponsors/ECF.svg" alt="Ethereum Community Fund" width="100px" />
+                </a>
+            </div>
+            <div class="column text-center">
+                <a href="https://www.ethereum.org/foundation" alt="Ethereum">
+                    <img src="./assets/sponsors/Ethereum.svg" alt="Ethereum Foundation" width="100px" />
+                </a>
+            </div>
+            <div class="column text-center">
+                <a href="https://spankchain.com" alt="Spank Chain">
+                    <img src="./assets/sponsors/SpankChain.svg" alt="Spank Chain" width="100px" />
+                </a>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section bg-grey-lightest">
+    <div parallax class="container">
+        <h1 class="title">Donations</h1>
+        <h2 class="subtitle">
+            <p>
+                Your support means a lot to us. With your help, we're able to accelerate our work on sharding,
+                furthering our goal to scale
+                the Ethereum blockchain for all. We have set up a multisig smart contract to accept donations.
+            </p>
+            <div class="flex justify-center p-3">
+                <div class="field has-addons ">
+                    <p class="control">
+                        <span class="button is-static is-large">Address: </span>
+                    </p>
+                    <p class="control">
+                        <input class="input is-large" type="text" value="0x9B984D5a03980D8dc0a24506c968465424c81DbE" />
```

Comment:
This MUST be immutable. Please do not use an input field to display the wallet address.

Reply:
absolutely right, i was playing around with the design and totally missed replacing the input with a label.